apps/studio/components/layouts/ProjectLayout/ProjectLayout.tsx

```diff
@@ -119,12 +122,19 @@ const ProjectLayout = forwardRef<HTMLDivElement, PropsWithChildren<ProjectLayout
       router.pathname === '/project/[ref]' || router.pathname.includes('/project/[ref]/settings')
     const showPausedState = isPaused && !ignorePausedState
 
-    const [isClient, setIsClient] = useState(false)
+    const handleMobileMenu = () => {
+      setIsSheetOpen(true)
+    }
 
     useEffect(() => {
       setIsClient(true)
     }, [])
 
+    useEffect(() => {
+      if (showConnect === 'true' && connectDialogUpdate) setShowConnectDialog(true)
```

Comment:
can this not be in connect component?
why does it need to be in a layout?

also need to clean up, remove the param once it's opened so a param isn't hanging around.